aptos-move/move-examples/marketplace/sources/coin_listing.move

```diff
@@ -121,6 +123,35 @@ module coin_listing {
         listing
     }
 
+    public entry fun update_fixed_price<CoinType>(
+        seller: &signer,
+        object: Object<Listing>,
+        price: u64,
+    ) acquires FixedPriceListing {
+        let listing_addr = object::object_address(&object);
+
+        if (!exists<FixedPriceListing<CoinType>>(listing_addr)) {
+            abort (error::not_found(ENO_LISTING))
+        };
+
+        let fixedPriceListing = borrow_global_mut<FixedPriceListing<CoinType>>(listing_addr);
+
+        if (fixedPriceListing.price == price) {
+            abort (error::invalid_argument(ESAME_PRICE))
+        };
+
+        fixedPriceListing.price = price;
+
+        events::emit_listing_placed(
```

Comment:
Wonder if we want a different event for this kind of update, but this should be good for now.